core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -58,7 +80,12 @@ angular.module('oppia').controller(
         var stateName = $scope.activeSuggestion.change.state_name;
         var contentType = contentId.split('_')[0];
 
-        return `${contentType} section of "${stateName}" card`;
+        var commitMessage = `${contentType} section of "${stateName}" card`;
+        if ($scope.translationUpdated) {
+          commitMessage = commitMessage + ' (with edits)';
```

Comment:
Dropped capitalization in "With Edits" -- change to "with edits".

Reference: https://github.com/oppia/oppia/pull/12424#discussion_r612955809
/cc @seanlip

Reply:
Could you explain where this commit message gets stored/shown? If it's in the history tab for an exploration, then instead, at the point where you commit a suggestion to the exploration/etc. in the backend, use the new boolean flag you've made in the other PR to construct the commit message correctly there. If it's stored in the suggestion then don't do anything special because you now have the separate boolean flag which records that information.

Reply:
It is stored as a commit for the exploration. Hence, I updated the commit message created in accept_suugestion method in suggestion_services to change the commit message when the suggestion is edited.